### ISSUE TITLE
Font Size Problem in Vertical Buttons Card

### ISSUE THREAD

**Issue by Rikorose** [N-l1/lovelace-soft-ui#13]
Hi

thanks for this nice template!

I got an issue where the font size is not applied to the h1 text in the vertical buttons card. Both, the  the upper text (supposed to be 20px) and the smaller text (supposed to be 15 px) have 28px, which is a default I guess. Tested with firefox and chromium. Also the font is Roboto, not Helvetica.
![snippet](https://user-images.githubusercontent.com/16517898/85032512-758ecb00-b16f-11ea-8650-03eb2cdb30d2.png)

I am not sure where the problem is. Apparently the style is not directly applied to `ha-card`, but is a `card-mod` element on the same html level:
![snippet](https://user-images.githubusercontent.com/16517898/85032403-54c67580-b16f-11ea-9cbc-f560948ed98d.png)

latest button-card.js and card-mod.js is installed.

corresponding yaml code:
```yaml
              - content: >
                  # There are  {% if is_state('sensor.lights_on', '0') %}
                  currently no  {% else %}  {{states('sensor.lights_on')}}  {%
                  endif %} lights on
                style: |
                  ha-card {
                  --paper-card-background-color: 'rgba(11, 11, 11, 0.00)';
                  box-shadow: 2px 2px rgba(0,0,0,0.0);
                     }
                  h1 {
                    font-size: 15px;
                    font-weight: thin;
                    font-family: Helvetica;
                    letter-spacing: '-0.01em';
                  }
                type: markdown
```

**Comment by KTibow** [comment]
Hmm... Have you tried turning it off and on again?
(reload the tab)

**Comment by Rikorose** [comment]
Yes also restart home-assistant, I could also ask at the card-mod repo. Maybe they have an idea.

**Comment by N-l1** [comment]
Hi @Rikorose, this is a known issue with card-mod and HA 0.111. For more info, see [here](https://github.com/thomasloven/lovelace-card-mod/issues/52). A fix is on the way.

**Comment by N-l1** [comment]
This is the new code that would be updated soon, @KTibow would you mind testing if this would work on 0.110 or lower. Thanks!
``` yaml
# Example entry
# Enter what you want to display
content: |
  # Enter what you want to display here
style:
  .: |
    ha-card {
      --paper-card-background-color: none !important;
      box-shadow: none !important;
      height: 50px; 
    }
  ha-markdown:
    $: |
      h1 {
        font-size: 15px;
        font-weight: thin;
        font-family: Helvetica;
        letter-spacing: '-0.01em';
      }
type: markdown
```

**Comment by KTibow** [comment]
Maybe fix: https://github.com/thomasloven/lovelace-card-mod/issues/52#issuecomment-644767155

**Comment by N-l1** [comment]
Yes, that's what I meant. "Fix" as in an update for this repo 👍

**Comment by KTibow** [comment]
@N-l1 do you know of any plans to fix this without changing the whole config? I did a little poking around and couldn't find anything.

**Comment by N-l1** [comment]
No, unfortunately to my knowledge the markdown styling config has to be updated for 0.111.

**Comment by N-l1** [comment]
This issue has been fixed as part of commit 36ba05c

**Comment by KTibow** [comment]
Note: The style wasn't applying for me, even after doing the fix. I fixed it by changing:
```yaml
style: |
```
to
```yaml
style:
```
